packages/go/cypher/analyzer/analyzer.go

```diff
@@ -62,7 +80,7 @@ type ComplexityMeasure struct {
 	nodeLookupKinds map[string]graph.Kinds
 }
 
-func (s *ComplexityMeasure) onFunctionInvocation(node *model.FunctionInvocation) {
+func (s *ComplexityMeasure) onFunctionInvocation(stack *model.WalkStack, node *model.FunctionInvocation) error {
```

Comment:
I can't remember if it's a compiler warning or not but you may want to consider renaming "stack" to "_" since that param is unused.

Reply:
Patched up